app/controllers/qing_groups_controller.rb

```diff
@@ -0,0 +1,51 @@
+class QingGroupsController < ApplicationController
+  # authorization via cancan
+  load_and_authorize_resource
+
+  before_filter :prepare_qing_group, :only => [:create]
+
+  def new
+    @qing_group = QingGroup.new(:form_id => params[:id])
+    render(:form)
+  end
```

Comment:
I don't think we'll need a new action since this form can be rendered when the page first loads and thus doesn't need an AJAX call.